src/store/record.js

```diff
@@ -191,11 +191,14 @@ let recordStore = {
         }
     },
     actions: {
-        async fetchRecord(state, id){
+        async fetchRecord(state, options){
             state.commit("resetCurrentRecordHistory");
             recordQuery.queryParam = {
-                id: id
+                id: options.id
             };
+            if (options.token) {
+                client.setHeader(options.token);
```

Comment:
You will need to reset the headers once you are finished with the query.
You can use `client.initalizeHeader()` after the awaits.